tutorial/output-cache-open-ai/OutputCacheDallESample/GenerateImageSC.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Azure.AI.OpenAI;
+using Azure;
+using Redis.OM;
+using Redis.OM.Vectorizers;
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+
+namespace OutputCacheDallESample
+{
+    public static class GenerateImageSC
+    {
+        private static RedisConnectionProvider _provider;
+        public static async Task GenerateImageSCAsync(HttpContext context, string _prompt, IConfiguration _config)
+        {
+            string imageURL;
+
+            string endpoint = _config["AZURE_OPENAI_ENDPOINT"];
```

Comment:
Please fix potential warnings here , also other places:
> Converting null literal or possible null value to non-nullable type.

Reply:
fixed. will include in the next commit